### ISSUE TITLE
Add auto deletion of a raid

### ISSUE THREAD

**Issue by reynozeros** [KineticFox/loa-lfgBot#37]
- auto delete a raid after a set amount of time after the raid date (let's say 3 days or so?)

**Comment by KineticFox** [comment]
for that we need a datetime object instead of text field for date/time (doable). 
Also a background job who checks in intervals if a raid has expired. (not that easy, not sure pycord offers that ability)

Needs a lot of resaerch. At the moment i have not the ressources to invest, but feel fre to make sugestions or pull requests :)

**Comment by Reekfrin** [comment]
Since you are using the 'send' command from discord we might be able to add "delete_after=86400" as an arg to the thread creation message for the raids ('m = await chanell.send('A Wild Raid spawns, come and join', embed=embed)'). this should delete the message after 10 days, shouldn't it? not sure if the thread gets deleted right with it tho.

**Comment by KineticFox** [comment]
@Reekfrin Yes this is possible too. But then we are only deleting the message in the corresponding channel and don't delete all the DB entries.

**Comment by KineticFox** [comment]
Thx to  @reynozeros https://guide.pycord.dev/extensions/tasks#concept this can work

**Comment by Reekfrin** [comment]
> for that we need a datetime object instead of text field for date/time (doable). 

Since you mention this, are you thinking about adding a command for the raid leader to put a time and date? this way we can also think about a task function to ping the raid members 15 min before the start or something similiar.

**Comment by Reekfrin** [comment]
Push, maybe use the last send message in a raid thread to determine the auto delete. ping the raid members 7 days after last message, if nothing happens for 3 more days, the raid gets deleted
